src/components/flat-table/flat-table-row/flat-table-row.spec.js

```diff
@@ -771,6 +781,18 @@ describe("FlatTableRow", () => {
 
           expect(onClickFn).toHaveBeenCalled();
         });
+
+        it("then the component should have tabIndex of 0 even if firstColumnExpandable", () => {
```

Comment:
No it shouldn't, that defeats the point of the firstColumnExpandable prop!

Reply:
Yeah I see what the functionality is supposed to do here, I hadn't seen that it would apply to the expandable row as well